src/extensionConflicts.ts

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved..
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { commands, Extension, extensions, window } from 'vscode';
+
+// A set of VSCode extension ID's that conflict with our extension
+const conflictingIDs = [
+  'haaaad.ansible',
+  'sysninja.vscode-ansible-mod',
+  'tomaciazek.ansible',
+  'vscoss.vscode-ansible',
+  'zbr.vscode-ansible',
+];
```

Comment:
How is this list going to be maintained?

Reply:
I wonder if this could be moved to `package.json`

Reply:
There is no need to move this anywhere as I do not expect us to do much maintenance on it. Probably we will have to modify the list less than once a year. All the other extensions that have a similar method did the same.

Reply:
Structurally, it's more obvious when it's in some sort of a data file / metadata. Having to update a list rarely is a questionable excuse for suboptimal design decisions. If the metadata is in a wrong place, it's wrong regardless of when it's going to be updated because the problem in question is of a different category.
Anyway, I'm not going to block this PR on this.